node/src/chain_spec.rs

```diff
@@ -156,22 +213,20 @@ fn testnet_genesis(
 			changes_trie_config: Default::default(),
 		}),
 		pallet_balances: Some(BalancesConfig {
-			// Configure endowed accounts with initial balance of 1 << 60.
-			balances: endowed_accounts
+			balances: founder_allocation
 				.iter()
-				.cloned()
-				.map(|k| (k, 1 << 60))
+				.map(|x| (x.0.clone(), x.1.clone()))
```

Comment:
You can use a Tuple destructuring here:

```suggestion
				.map(|(accountId, balance)| (accountId.clone(), balance.clone()))
```